Documentation/community/governance/commit_access.rst

```diff
@@ -186,11 +186,11 @@ The process to grant commit access to a candidate is simple:
 
 -  If the candidate agrees access is granted by setting up commit access
    to the repos. The new committer is invited to the #committers Slack channel,
-   *after* the nomination poll and related discussions have been deleted. The
-   name of the new committer is also added to the list in the MAINTAINERS.md_
-   file. The new committer is also added to the CNCF's list of Cilium
-   maintainers as documented `here
-   <https://github.com/cncf/foundation/blob/main/README.md#other-content>`__. 
+   *after* the nomination poll and related discussions have been deleted. There
```

Comment:
Should we also explicitly state the motivation for deleting the poll result and related discussions?